pkg/client/results/processing.go

```diff
@@ -57,6 +57,18 @@ const (
 	// PostProcessedResultsFile is the name of the file we create when doing
 	// postprocessing on the plugin results.
 	PostProcessedResultsFile = "sonobuoy_results.yaml"
+
+	// metadataFileKey is the key used in an Item's metadata field when the Item is
+	// representing the a file summary (and its leaf nodes are individual tests or suites).
+	metadataFileKey = "file"
+
+	// metadataTypeKey is the key used in an Item's metadata field when describing what type
+	// of entry in the tree it is. Currently we just tag summaries, files, and nodes.
+	metadataTypeKey = "type"
+
+	metadataTypeNode    = "node"
+	metadataTypeFile    = "file"
+	metadataTypeSummary = "plugin-summary"
```

Comment:
Is this the name we want here? The other values (node/file) seem right but I'm not sure about this one.

Reply:
I think just “summary” alone might be enough?